backend/src/schema/types/type/User.gql

```diff
@@ -23,6 +23,8 @@ type User {
   #updatedAt: DateTime
   createdAt: String
   updatedAt: String
+  termsAndConditionsConfirmedAt: String
```

Comment:
```suggestion
```

We need `termsAndConditionsConfirmedVersion` to compare it with the current version of terms and conditions, but we don't need to expose `termsAndConditionsConfirmedAt`.